spec/infinispan_local_spec.js

```diff
@@ -183,6 +201,101 @@ describe('Infinispan local client', function() {
           {'includeState' : true}))
       .catch(t.failed(done));
   });
+  it('fails when wrong event name is passed', function(done) { client
```

Comment:
If you expect the test to fail, the `.catch` call should inspect the error and verify that it's expected error and call `done()`. As it stands, if there's an error being thrown here, it will call `done(error)` which will make the test fail. And the finally section should be adjusted too. So if no errors were produced, done should be called with `done(new Error("Ups, you should have succeeded"))`.